WalletWasabi/Blockchain/BlockFilters/IndexBuilderService.cs

```diff
@@ -75,18 +75,18 @@ public IndexBuilderService(IndexType indexType, IRPCClient rpc, BlockNotifier bl
 
 	public static byte[][] DummyScript { get; } = new byte[][] { ByteHelpers.FromHex("0009BBE4C2D17185643765C265819BF5261755247D") };
 
-	public IRPCClient RpcClient { get; }
-	public BlockNotifier BlockNotifier { get; }
+	private IRPCClient RpcClient { get; }
+	private BlockNotifier BlockNotifier { get; }
 	public string IndexFilePath { get; }
```

Comment:
What about this one?

```suggestion
	private string IndexFilePath { get; }
```